src/modules/Packages.rb

```diff
@@ -1269,9 +1269,15 @@ def CheckContentFile(source)
         )
         return true
       end
+      ramdisk = Convert.to_string(SCR.Read(path(".target.string"), "/content"))
+      if ramdisk == nil
```

Comment:
Maybe a simple `if !File.exists?("/content")` would be better here. This legacy code is a bit cryptic...